docs/cip/README.md

```diff
@@ -4,704 +4,2800 @@ Title: Ouroboros Leios - Greater transaction throughput
+while maintaining decentralization.
+
+While the empirical evidence demonstrates Leios's performance capabilities, any
+protocol modification introduces new attack vectors and operational constraints
+that must be carefully assessed. The following section examines potential
+security risks and practical constraints that inform deployment considerations.
+
+### Trade-offs & Limitations
+
+High-throughput blockchain protocols require fundamental trade-offs between
+throughput, latency, complexity, and security. The proposed specification
+balances these competing dimensions to achieve substantial throughput gains
+while preserving ecosystem compatibility.
+
+<div align="center">
+<a name="figure-19" id="figure-19"></a>
+
+![Leios Variants Comparison](images/leios-variants-comparison-radar.svg)
+
+<em>Figure 19: Comparison: Praos (red), proposed Leios (teal), and research
+Leios (orange)</em>
+
+</div>
+
+The comparison illustrates the fundamental trade-off between throughput capacity
+and ecosystem disruption. While the research paper's approach (orange) achieves
+higher throughput, it requires extensive ecosystem changes, significantly longer
+confirmation times (2-3 minutes), and extended development timelines (2.5-3
+years). The proposed specification (teal) strikes a strategic balance,
+delivering substantial throughput improvements while maintaining manageable
+latency increases and ecosystem compatibility.
+
+**Key Trade-offs:**
+
+- **Throughput**: 30-65x increase (from ~4.5 TxkB/s to ~140-300 TxkB/s)
+  addresses economic sustainability and Reserve depletion timeline
+- **Transaction Inclusion Latency**: Increases from ~20 seconds to 45-60 seconds
+- **Infrastructure**: Requires modest hardware upgrades (4-core machines) and
+  cryptographic key registration, offset by enhanced rewards
+- **Time to Market**: ~1-1.5 years for deployment versus 2.5-3 years for
+  higher-concurrency alternatives
+
+**Protocol Limitations:**
+
+The specification involves operational constraints including timing dependencies
+that require careful parameter tuning, confirmation complexity through multiple
+transaction states, and interdependent protocol parameters requiring ongoing
+optimization. These represent acceptable trade-offs for achieving substantial
+throughput improvements while maintaining familiar transaction structures and
+ecosystem compatibility.
+
+**Security Considerations:**
+
+The proposed specification maintains Praos security properties while introducing
+minimal new attack vectors. New threats include equivocation by EB
+producers/voters, transaction availability attacks, and denial-of-service
+through invalid transactions. These are mitigated through cryptographic
+validation, equivocation detection, redundant connections, and economic
+disincentives. Comprehensive analysis is documented in the
+[threat model](https://github.com/input-output-hk/ouroboros-leios/blob/main/docs/threat-model.md)
```

Comment:
Throughout this document, we should consider using permanlinks instead of ephemeral ones.